core/array.cpp

```diff
@@ -334,6 +335,115 @@ Array Array::slice(int p_begin, int p_end, int p_step, bool p_deep) const { // l
 	return new_arr;
 }
 
+Array Array::filter(const Callable &p_callable) const {
+	Array new_arr;
+
+	for (int i = 0; i < size(); i++) {
+		const Variant *arg = &get(i);
+		Variant result;
+		Callable::CallError ce;
+		p_callable.call(&arg, 1, result, ce);
+
+		if (ce.error != Callable::CallError::CALL_OK) {
+			switch (ce.error) {
```

Comment:
You need to have a case for every value in the enum, even if it does nothing. You can use `default` too, but since there's only two elements remaining, I prefer to add them explicitly. So if someone add a new error value in the future, it'll break the build here and they'll have to handle it here too.

Reply:
BTW, this applies to all functions.